src/client/src/app/lib/components/feedback-banner.tsx

```diff
@@ -0,0 +1,24 @@
+import Tag from '@components/nhsuk-frontend/tag';
+import '../../global.css';
+
+const FeedbackBanner = () => {
+  return (
+    <div className="feedback-banner">
+      <Tag text="Feedback" />
+      &nbsp;
```

Comment:
I don't think it's likely to cause problems in our site, but in general we shouldn't be using `&nbsp;` for non-breaking spaces in tsx as it can insert unexpected tags (e.g. in some browsers this becomes `<span></span>` which could cause unexpected changes if we had a css rule targetting span elements)
Just `{'  '}` is the more modern way to handle this and how we're doing it elsewhere